opensource/index.md

```diff
@@ -1,17 +1,95 @@
 ---
 description: Overview of contributing
 keywords: open, source, contributing, overview
-title: Open Source at Docker
+title: Contribute to documentation and open source projects
 redirect_from:
 - /contributing/
 - /contributing/contributing/
+- /opensource/code/
+- /opensource/project/who-written-for/
+- /opensource/project/software-required/
+- /opensource/project/software-req-win/
+- /opensource/project/set-up-git/
+- /opensource/project/set-up-dev-env/
+- /opensource/project/test-and-docs/
+- /opensource/project/who-written-for/
+- /opensource/workflow/advanced-contributing/
+- /opensource/workflow/
+- /opensource/workflow/coding-style/
+- /opensource/workflow/create-pr/
+- /opensource/workflow/find-an-issue/
+- /opensource/workflow/make-a-contribution/
+- /opensource/workflow/review-pr/
+- /opensource/workflow/work-issue/
+- /opensource/FAQ/
+- /opensource/get-help/
+- /opensource/code/
+- /opensource/doc-style/
+- /opensource/governance/
+- /opensource/governance/dgab-info/
+- /opensource/governance/board-profiles/
+- /opensource/governance/conduct-code/
+- /opensource/ways/
+- /opensource/ways/meetups/
+- /opensource/ways/issues/
+- /opensource/ways/community/
+- /opensource/ways/test/
+
 ---
 
-Contributing to the Docker project or to any open source project can be a
-rewarding experience. You help yourself and you help the projects you work on.
-You also help the countless number of other project participants.
+Contributing to the Docker documentation can be a rewarding experience. When you
+offer feedback, questions, edits, or new content, you help us, the projects you
+work on, and the larger Docker community.
+
+We welcome your particpation to help make the documentation better!
+
+## How to contribute to the docs
+
+The Docker product documentation is published at
+[docs.docker.com](https://docs.docker.com/).
+
+There are many ways to contribute:
+
+- Edit, rate, or file an issue or question directly on the site by
+using the links available on the right-side menu on every page
+at [docs.docker.com](https://docs.docker.com/).
+
+  ![Docs feedback links](images/docs-site-feedback.png)
+
+- File a documentation issue on GitHub at
```

Comment:
Maybe say that this is the same thing as clicking the *Request doc changes* link on a given page?